### PR TITLE
Explaining the why and how to contribute

### PR DESCRIPTION
**Pull request by mbbroberg**
* Temporary update to README to give context
* Using default community template for CONTRIBUTING.md
* Partially addresses #90

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Contributing
+
+Thank you for wanting to contribute back to the Sensu Community! Take a look through here to be ready to get your PR quickly reviewed 👌.
+
+## Guidelines
+We ask that you keep the following guidelines in mind when planning your contribution:
+
+* Whether your contribution is for a bug fix or a feature request, **create an [Issue](https://github.com/sensu-plugins/sensu-docs-site/issues)** to let us know what you are thinking before you fix it. It helps us give a LGTM much faster (with fewer cases of saying no to a PR)
+* **For bugs**, if you have already found a fix, feel free to submit a Pull Request referencing the Issue you created. Include the `Fixes #` syntax to link it to the issue you're addressing.
+* **For feature requests**, we want to improve upon the library incrementally which means small changes at a time. In order to ensure your PR can be reviewed in a timely manner, please keep PRs small. If you think this is unrealistic, then mention that within the issue and we can discuss it.
```
Comment (cwjohnston) [inline CONTRIBUTING.md]: I think this should read "improve upon the documentation incrementally" instead of "the library"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+$ git clone https://github.com/sensu-plugins/sensu-docs-site/
+# or: git clone git@github.com:$user/sensu-docs-site.git
+
+$ cd sensu-docs-site
+
+$ git remote add upstream https://github.com/sensu-plugins/sensu-docs-site.git
+# or: git remote add upstream git@github.com:sensu-plugins/sensu-docs-site.git
+
+# Never allow a push to upstream master
+$ git remote set-url --push upstream no_push
```
Comment (cwjohnston) [inline CONTRIBUTING.md]: I didn't know about this. Neat!

File: README.md [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+# Running Sensu Docs
 
-    - [Variables and Accessing Front Matter](#variables-and-accessing-front-matter)
-
-    - [Search](#search)
-
-## Project Setup
+Now that our project can be run locally with Hugo, there's some new steps to learn. Don't worry, we've documented how to do so:
 
 ### Getting Started with Hugo
```
Comment (cwjohnston) [inline README.md]: Can we please replace platform specific setup instructions with a link to https://gohugo.io/getting-started/installing or similar? I don't think we benefit from duplicating that sort of information here.

**Comment by cwjohnston** [comment]
Thanks @mbbroberg ! I added some review comments, and it looks like we need a rebase as well.

File: README.md (hunk 3 above)
Comment (mbbroberg) [reply README.md]: I haven't dug into the nuances of the setup so I don't want to delete anything until I know we have it covered elsewhere. Issue #144 is intended to resolve this.

**Comment by mbbroberg** [comment]
Thanks for the review @cwjohnston! I made the minor edit requested but will pass on the one that is out of scope for this PR.

**Pull request merged**